index.html

```diff
@@ -202,35 +211,48 @@
         ENTER: function(e, original) {
           line = this.get_command();
           if (multilineMode.enabled) {
-            if (multilineMode.mode === ML_FUNC) {
-              /*
-              *   We captured the 'enter' key, so to get the regex right we have
-              *   to spoof adding in the newline.
-              */
-              let tempLine = line + "\n";
-              if (tempLine.match(/(\.{3}\s+\n){2}/gm)) {
-                multilineMode.enabled = false;
-                /*
-                *   This will run the callback to terminal() and then the entire 
-                *   statement will be passed in for processing.
-                */
-                return original();
-              } else {
-                this.insert(multilineFuncGutter);
-              }
-            } else if (multilineMode.mode === ML_STRUCT) {
-              if (line.match(/[}\]\)]$/)) {
+            switch (multilineMode.mode) {
```

Comment:
I don't like switches for stuff with if...else logic nested inside. The `break` statement thing can easily get removed and cause nasty fall-through issues.

Reply:
Would you like me to convert it back to an if-else statement?

Reply:
Yeah, I think it's safer as if...else.

Reply:
Ok, I converted it https://github.com/daniel-schroeder-dev/cwhq-python-repl-hacking/pull/1/commits/f718a52e76105abc9d6238d0cf450818c6da71d0.